Orange/tests/test_variable.py

```diff
@@ -247,6 +278,16 @@ def test_val(self):
         self.assertEqual(a.str_val(Value(a, "")), "?")
         self.assertEqual(a.repr_val(Value(a, "foo")), '"foo"')
 
+    def test_copy_copies_attributes(self):
+        var = StringVariable("x")
+        var.attributes["a"] = "b"
+        var2 = var.copy(compute_value=None)
+        self.assertIn("a", var2.attributes)
+
+        var2.attributes["a"] = "c"
+        # Attributes of original value should not change
+        self.assertEqual(var.attributes["a"], "b")
```

Comment:
This one, parameterized. Instead of [line 213](https://github.com/biolab/orange3/pull/1272/files/eb077a89b476e6fb7bddf2ee731c5d0e88123dec#diff-f2a8cf0957f5b1acd3c33a160932064bR213) and [line 261](https://github.com/biolab/orange3/pull/1272/files/eb077a89b476e6fb7bddf2ee731c5d0e88123dec#diff-f2a8cf0957f5b1acd3c33a160932064bR261) and [line 281](https://github.com/biolab/orange3/pull/1272/files/eb077a89b476e6fb7bddf2ee731c5d0e88123dec#diff-f2a8cf0957f5b1acd3c33a160932064bR281).